website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+  Packer can be configured to publish artifact metadata for all completed builds to an active HCP Packer registry.
+  The configuration for HCP Packer is managed by environment variables for legacy JSON templates or via a hcp_packer_registry
+  block for HCL2 templates. 
+page_title: Configuring Packer for HCP Packer
```

Comment:
I think we could just go with "Configuring HCP Packer " - I feel like the extra Packer is a bit confusing

Reply:
💯 thanks for calling that out.